### ISSUE TITLE
bn

### ISSUE THREAD

**Issue by thudzj** [Harry24k/bayesian-neural-network-pytorch#4]
Hi,

Here is a bug: https://github.com/Harry24k/bayesian-neural-network-pytorch/blob/master/torchbnn/modules/batchnorm.py#L115.

**Comment by Harry24k** [comment]
Thanks!
